### ISSUE TITLE
Group Skipper E2E Tests

### ISSUE THREAD

**Issue by Zeyber** [SAP/cloud-commerce-spartacus-storefront#5389]

**Comment by Zeyber** [comment]
QA:
- Test `group-skipping.e2e.spec.ts` runs and passes

**Comment by KateChuen** [comment]
Ran the test on dev3, with dev3 backend.
It failed at checkout because of the total not matching.
![checkout-dev3](https://user-images.githubusercontent.com/45409515/74849823-30fd3b00-5307-11ea-8d28-ff3909630f24.png)



Ran the test on dev3, against dev12 backend.
It also failed at checkout, but it's because it never logged the user in.
![checkout](https://user-images.githubusercontent.com/45409515/74849836-33f82b80-5307-11ea-95ca-fca38a991d4d.png)

**Comment by kacperknapik** [comment]
**QA results**
- used dev12
- fixed one small error

Works fine now
